src/ensembl/src/content/app/entity-viewer/state/sidebar/entityViewerSidebarState.ts

```diff
@@ -1,27 +1,65 @@
 import { Status } from 'src/shared/types/status';
+import { Gene } from '../../types/gene';
+import { Transcript } from '../../types/transcript';
+import { Assembly } from '../../types/assembly';
+import { DataSet } from '../../types/dataSet';
+import { Homeologue } from '../../types/homeologue';
+import { Publication } from '../../types/publication';
+import JSONValue from 'src/shared/types/JSON';
 
 export enum SidebarTabName {
-  OVERVIEW = 'overview',
-  PUBLICATIONS = 'publications'
+  OVERVIEW = 'Overview',
+  EXTERNAL_REFERENCES = 'External references'
 }
 
 export type SidebarStatus = Status.OPEN | Status.CLOSED;
 
+type GeneViewTranscriptSidebarPayload = Pick<Transcript, 'id' | 'xrefs'>;
+
+// TODO: Not sure if this is the best approach as the sidebar doesn't need any other data other than the ones picked below.
```

Comment:
General consideration: if we continue accessing the graphql server (Thoas) directly from the client and if we use apollo-client for doing that, the sidebar data will not even get to be saved to redux, but will be accessed directly from inside the component that issues the query to apollo-client.

Reply:
Does it mean the data will be retrieved and transformed every time we re-render?

Reply:
No, apollo-client has a cache; if the query is the same when the component rerenders, apollo-client just returns the value it's already received from the backend.